lib/ansible/modules/packaging/os/apt.py

```diff
@@ -10,9 +10,11 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['stableinterface'],
-                    'supported_by': 'core'}
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['stableinterface'],
+    'supported_by': 'core'
+}
```

Comment:
Please remove this, having this toggle in modules you don't own is going to get annoying...